openapi/components/schemas/AmlSettings.yaml

```
@@ -95,6 +95,46 @@ properties:
             $ref: ./AmlConfidence.yaml
           noCountry:
             $ref: ./AmlConfidence.yaml
+      cityAndRegionMatch:
+        type: object
+        description: Exact match of the customer's city and region of residence and exact name is found.
```

suggestion: for readability
```suggestion
        description: Exact match of the customer's city, region of residence, and name is found.
```